attestation-container/attestation-container.go

```diff
@@ -40,25 +42,30 @@ func (s *server) FetchAttestation(ctx context.Context, in *pb.FetchAttestationRe
 		return nil, status.Errorf(codes.Internal, "failed to fetch attestation report: %s", err)
 	}
 
-	var endorsement []byte
+	var attestationEndorsement []byte
 	if endorsementEnvironmentValue == nil {
 		reportedTCBBytes := reportBytes[attest.REPORTED_TCB_OFFSET : attest.REPORTED_TCB_OFFSET+attest.REPORTED_TCB_SIZE]
 		chipIDBytes := reportBytes[attest.CHIP_ID_OFFSET : attest.CHIP_ID_OFFSET+attest.CHIP_ID_SIZE]
-		endorsement, err = attest.FetchAttestationEndorsement(*endorsementServer, reportedTCBBytes, chipIDBytes)
+		attestationEndorsement, err = attest.FetchAttestationEndorsement(*attestationEndorsementServer, reportedTCBBytes, chipIDBytes)
 		if err != nil {
 			return nil, status.Errorf(codes.Internal, "failed to fetch attestation endorsement: %s", err)
 		}
 	} else {
-		endorsement = append(endorsement, endorsementEnvironmentValue.VcekCert...)
-		endorsement = append(endorsement, endorsementEnvironmentValue.CertificateChain...)
+		attestationEndorsement = append(attestationEndorsement, endorsementEnvironmentValue.VcekCert...)
+		attestationEndorsement = append(attestationEndorsement, endorsementEnvironmentValue.CertificateChain...)
 	}
 
-	return &pb.FetchAttestationReply{Attestation: reportBytes, AttestationEndorsementCertificates: endorsement}, nil
+	uvmEndorsement, err := uvm.FetchUVMEndorsement(*uvmEndorsementEnvVar)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "failed to fetch UVM endorsement: %s", err)
+	}
```

Comment:
Similar to `attestationEndorsement`, I think we should parse the environment variable once and store it in a global variable on startup rather than for each request?

Reply:
done